esp/esdllib/README.md

```diff
@@ -546,33 +648,20 @@ Each of these parameters gives the script access to some information about the c
 <es:param name="clientversion" select="'unknown'"/>
 ```
 
-If the parameter is undefined, then it is assigned the value of the `@select` XPath expression.
+If the parameter is undefined, then it is assigned the value of the `@select` XPath expression. These are the available system parameters:
 
 * `$clientversion` : The client interface version specified in the incoming request.
 * `$espUserName` : The username of the user authenticated to run the current transaction.
 * `$espUserPeer` : The IP address of host originating the current transaction.
 * `$espUserRealm` : The 'Company' or parent entity to the currently authenticated user.
-* `$espUserStatus` : The status of the currently authenticated user (eg *Active*, *Suspended*, etc., as defined by the security plugin and back end in use)
+* `$espUserStatus` : The status of the currently authenticated user (eg *Active*, *Suspended*, etc., as defined by the security plugin and backend in use)
```

Comment:
Missing `$espUserStatusString`. Where `$espUserStatus` is the numeric value assigned in an enumeration, `$espUserStatusString` is a human readable equivalent. One may choose to code to the string to avoid possible (albeit unlikely) compatibility issues with an altered enum.

Reply:
Added espUserStatusString and table giving the values for both functions